frontend/app/components/wp-relations/wp-relations-create/wp-relations-create.directive.ts

```diff
@@ -0,0 +1,117 @@
+import {wpDirectivesModule} from '../../../angular-modules';
+import {WorkPackageResourceInterface} from '../../api/api-v3/hal-resources/work-package-resource.service';
+import {RelationType} from "../wp-relations.interfaces";
+
+export class WpRelationsCreateController {
```

Comment:
Always use full words for class names.
So it should be `WorkPackageRelationsCreateController`.